scripts/upgradeable/transpile.sh

```diff
@@ -6,7 +6,7 @@ VERSION="$(jq -r .version contracts/package.json)"
 DIRNAME="$(dirname -- "${BASH_SOURCE[0]}")"
 
 bash "$DIRNAME/patch-apply.sh"
-sed -i "s/<package-version>/$VERSION/g" contracts/package.json
+sed -i'' -e "s/<package-version>/$VERSION/g" "contracts/package.json"
```

Comment:
can you explain this change ?

Reply:
Yes, it's a version that works with both Linux and Mac's `sed`.

More [here](https://stackoverflow.com/a/4247319).

Reply:
I learned something today ... and it reinforces my believe that true devs are not/shoud not be using mac

Reply:
I'll be happy not being a true dev, thanks!

Reply:
Your using a mac ? All my condolences (particularly on being stuck with a non standard keyboard)

Reply:
Not a problem on my side. I honestly don't care what you're using as long as we keep a workspace that works for both